plugin/pkg/scheduler/factory/plugins.go

```diff
@@ -356,9 +358,27 @@ func getPriorityFunctionConfigs(names sets.String, args PluginFactoryArgs) ([]al
 			})
 		}
 	}
+	var totalPriority int
+	for _, config := range configs {
+		// Checks for overflow and returns errors.
+		// TODO: Need to check overflow for weight*10.
```

Comment:
Right but we need to have check for * but that's for the calling function.(Means if we have a unit test for getPriorityFunctionConfigs.